apps/bmd/src/pages/PrintOnlyScreen.tsx

```diff
@@ -75,6 +74,8 @@ const PrintOnlyScreen: React.FC<Props> = ({
     !isCardVotesEmpty &&
     !isPrinted
 
+  useEffect(useEffectToggleLargeDisplay, [ballotStyleId])
```

Comment:
Why `ballotStyleId`? Will this not trigger properly if we get two consecutive cards with the same ballot style?

Reply:
The PrintOnlyScreen is kind of a meta screen which has many views (Perhaps it should be renamed to `PrintOnlyAppScreen`?) This screen should always use the larger font size, but needs to run any time a card is present. Given this explanation, I'm changing this line to:
```
useEffect(useEffectToggleLargeDisplay, [isVoterCardPresent])
```